deployment/portable/.env.example

```
@@ -0,0 +1,49 @@
+GOOGLE_CLIENT_SECRET=
+GOOGLE_WEB_LOGIN_ENABLED=true
+KEYCLOAK_WEB_ENABLED=false
+KEYCLOAK_IMAGE=quay.io/keycloak/keycloak:26.0
+KEYCLOAK_PUBLIC_BASE_URL=http://localhost:8082
+KEYCLOAK_INTERNAL_BASE_URL=http://keycloak:8080
+KEYCLOAK_REALM=xyne-shared
+KEYCLOAK_CLIENT_ID=xyne-web
+KEYCLOAK_CLIENT_SECRET=
+KEYCLOAK_WORKSPACE_EXTERNAL_ID=
+KEYCLOAK_LOGOUT_REDIRECT_URL=/auth
+KEYCLOAK_ADMIN=admin
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The KEYCLOAK_ADMIN key should go before the KEYCLOAK_CLIENT_ID key

(UnorderedKey)